requirements.txt

```diff
@@ -15,7 +15,7 @@ scikit-image>=0.14.0,<0.17.0
 scikit-build<1.0.0
 statsmodels==0.9.0
 simpleitk<2.0.0
-argschema<2.0.0
+argschema==2.0.2
```

Comment:
Should this be `argschema < 3.0.0`?

Or do we anticipate problems with any version larger than 2.0.2?

Reply:
Sure, I will update to `< 3.0.0`